config/rubocop/1.9.1/rubocop-2.7.yml

```diff
@@ -82,6 +85,9 @@ Naming/MethodParameterName:
 Naming/RescuedExceptionsVariableName:
   Enabled: false
 
+Naming/VariableNumber:
+  Enabled: false
```

Comment:
Don't really have a standard, we have symbols that conform to both styles like `:Address1` and `:desktop_col_3`.